utilities_help/createToolboxMLTBX.m

```diff
@@ -5,7 +5,8 @@ function createToolboxMLTBX(verNumber)
 % project root.
 % Input:
 %   verNumber- string in major.minor.bugfix format
-prjFileName=fullfile("fsdaToolboxPackaging.prj");
+% Copyright 2023 The MathWorks, Inc.
+prjFileName="fsdaToolboxPackaging.prj";
 toolboxOption=matlab.addons.toolbox.ToolboxOptions(prjFileName);
```

Comment:
Can we verify if the fsdaToolboxPackaging.prj is present at the location and error out if not

Reply:
Moved fsdaToolboxPackaging.prj  as a input argument, added code to check the prjfile is a string.